### DISCUSSION TITLE
migrating-to-cloud-native-application-architectures/the-rise-of-cloud-native/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#99] [Announcements]
# migrating-to-cloud-native-application-architectures/the-rise-of-cloud-native/

一站式云原生资料库

https://lib.jimmysong.io/migrating-to-cloud-native-application-architectures/the-rise-of-cloud-native/

**Comment by small-fresh-meet** [comment]
这些电子书是否可以下载

**Reply by rootsongjc** [reply]
你好，目前未提供下载，因为很多书（小册子）只有几个页面，你可以自己制作 PDF。
